app/templates/dashboard.html

```diff
@@ -24,58 +24,57 @@ <h1 class="govuk-heading-xl">Your applications</h1>
         {% endset %}
 
         {% if not application.status == 'SUBMITTED' %}
-            {% set application_link = '<a class="govuk-link" href="{}">Continue application</a>'.format(url_for('routes.tasklist', application_id=application.id)) %}
+            {% set application_link = '<a class="govuk-link" href="{}">{}</a>'.format(url_for('routes.tasklist', application_id=application.id), gettext('Continue application')) %}
         {% endif %}
 
+        {% set row = ns.rows.append(
+        [
+            {
+                'text': application.project_name if application.project_name else gettext('Untitled project')
+            },
+            {
+                'text':  govukTag({
+                    'text': application.status|snake_case_to_human ,
```

Comment:
```suggestion
                    'text': application.status.name|snake_case_to_human ,
```

This is an Enum so need to do `.name` or the UI is throwing an error for me

Reply:
Also - should we find a way to translate the statuses? Seems since they're based on the Enum, and being formatted using `snake_case_to_human` they will always be English

Reply:
Disregard this, it's actually a string.  I had mocked an Enum without realizing it wasn't being used